helloflask_flask-dropzone_examples_complete-redirect_app_20.py

```
@@ -0,0 +1,43 @@
+        f = request.files.get('file')
+        f.save(os.path.join(app.config['UPLOADED_PATH'], f.filename))
+    return render_template('index.html')
+
+
+@app.route('/completed')
+def completed():
+    return '<h1>The Redirected Page</h1><p>Upload completed.</p>'
+
+
+if __name__ == '__main__':
+    app.run(debug=True)
```

Ruff (0.13.1)

Use of `debug=True` in Flask app detected

(S201)